Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -520,6 +518,7 @@
 
 		%% 4.c Load data
         Data3D = []; % initialize empty variable
+		Data4D = []; % initialize empty variable

```

Comment:
The way I would write it, we always have Data4D (which can be either 3D or 4D, depending on the input), which would get us cleaner code. 

I agree that the way you started this is quicker for now, but we could revamp this later. So fine for now

Reply:
Agree. Lets do the in the follow-up issues. and close this now.

Reply:
New issue #1849 is created